src/screens/Ticketing/Ticket/Details/QrCode.tsx

```diff
@@ -179,6 +216,33 @@ const QrCodeLoading = () => {
   );
 };
 
+const StaticQr = ({fc}: {fc: FareContract}) => {
+  const styles = useStyles();
+  const {t} = useTranslation();
+  const [qrCodeSvg, setQrCodeSvg] = useState<string>();
+
+  useEffect(() => {
+    if (fc.qrCode) {
+      qrcode.toString(fc.qrCode, {type: 'svg'}).then(setQrCodeSvg);
+    }
+  }, [fc.qrCode, setQrCodeSvg]);
```

Comment:
why does **setQrCodeSvg** have to be in the dependency array here? 🤔

Reply:
This code was there from before and just moved by me. It doesn't have to be, but we haven't any coding guideline regarding this for our projects.

I tried to find if there was any best-practice regarding this, but nothing clear cut. Quote [Dan Abramov](https://github.com/reactjs/reactjs.org/issues/4506):
> … you can add it for consistency but it doesn't really matter one way or another.

Reply:
hmmm.. I atleast dont really do that. 😅